src/App.js

```diff
@@ -8,6 +8,8 @@ import { Route, Routes } from "react-router-dom";
 import { SignIn } from "./Components/SignIn/SignIn";
 import { LandingPage } from "./Pages/LandingPage/LandingPage";
 import { useAuth } from "./Context/loginContext";
+import { SignUp } from "./Components/SignUp/SignUp";
+import { Archive } from "./Pages/Archive/Archive";
```

Comment:
you could import from a single index file for common paths - components, and pages.